README.md

```diff
@@ -1,5 +1,10 @@
-# ATLAS animal_data Service Template \[v0.1.1\]
+# ATLAS animal_data Service Template \[v0.2\]
 
 [Documentation](https://htmlpreview.github.io/?https://github.com/atlasH2020-templates/animal_data/blob/v0.1.1/doc.html)
 
 [OpenAPI Specifications](https://sensorsystems.iais.fraunhofer.de/doc/?url=https://raw.githubusercontent.com/atlasH2020-templates/animal_data/v0.1.1/oas)  
+
+## Branch specific links
```

Comment:
just temporarily, remove before merge.

Reply:
version in url (also for OAS) should be v0.2.0 as well